Getting-Started.md

```diff
@@ -39,15 +39,32 @@ Step 3: Ensuring your RWS meets the technical requirements
 Step 4: Testing your RWS locally
 --------------------------------
 
-Once you've made your changes to your local branch, you can open a terminal and run the command `python3 check_sites.py`. Currently, this will produce a number of check failures due to older submissions in the set failing recent changes to checks. If no members of your set are mentioned in the list of failed checks, then your submission should be fine. If you would like a cleaner way of looking for problems in your set, follow these steps:
+Once you've made your changes to your local branch, you can open a terminal and run the command 
 
-1.  Run `cp related_website_sets.JSON my_rws.JSON` in your shell
+`python3 check_sites.py --primaries=<your primary site>`
```

Comment:
Personal preference, I would make this example concrete as well, since I expect most people will only read down to here.

Reply:
Done.